tests/rt.conf

```diff
@@ -252,20 +252,18 @@ RUN     | cpld_debug
 #######################################################################################################################################################################################
 # Data Atmosphere tests
 #######################################################################################################################################################################################
+COMPILE | DATM=Y S2S=Y                                                                                                        | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
```

Comment:
Is there a reason you've removed the comment-ed out Cheyenne lines?

Reply:
It will be added.